addons/mail/models/mail_message.py

```diff
@@ -1044,7 +1044,7 @@ def _to_store(
                 "write_date": message.write_date,
                 "is_note": message.subtype_id.id == note_id,
                 "is_discussion": message.subtype_id.id == com_id,
-                "subtype_description": message.subtype_id.description,
+                "subtype_description": message.subtype_id.sudo().description,
```

Comment:
`mail_message_subtype_rule_public` restrict public user from reading internal `subtype`

This makes sense, and we should not leak internal message to public.
This might be from an earlier version of the PR were the default subtype (note) was used when posting even in portal. This might be fixed.

Reply:
Indeed it is a wanted security rule :) so I don't think we want to leak that information, as portal should never read internal messages, nor post them obviously.